x/distribution/keeper/store.go

```diff
@@ -179,50 +179,6 @@ func (k Keeper) GetValidatorHistoricalReferenceCount(ctx context.Context) (count
 	return
 }
 
-// get current rewards for a validator
-func (k Keeper) GetValidatorCurrentRewards(ctx context.Context, val sdk.ValAddress) (rewards types.ValidatorCurrentRewards, err error) {
```

Comment:
There's a small behavioural change for which:
- Before: if reward did not exist, we do not error but return un-initialised types.ValidatorCurrentRewards
- Now: if reward does not exist we error.